misc/probe2trace.pl

```diff
@@ -110,6 +136,22 @@
             } else {
                 push @ap, map{"arg${i}->$_"} qw(minimum smoothed latest);
             }
+        } elsif ($type eq 'struct st_quicly_stats_t *') {
+            push @fmt, map {qq("$_":\%u)} qw(min_rtt smoothed_rtt rttvar);
+            push @fmt, map {qq("$_":\%u)} qw(cwnd ssthresh);
+            push @fmt, map {qq("$_":\%llu)} qw(pkts_out packets_acked loss_total loss_spurious pkts_in decryption_failed);
+            push @fmt, map {qq("$_":\%llu)} qw(total_bytes_out total_bytes_in);
+            if ($arch eq 'linux') {
+                push @ap, map{"((struct st_quicly_stats_t *)arg$i)->rtt.$_"} qw(minimum smoothed variance);
+                push @ap, map{"((struct st_quicly_stats_t *)arg$i)->cc.$_"} qw(cwnd ssthresh);
+                push @ap, map{"((struct st_quicly_stats_t *)arg$i)->num_packets.$_"} qw(sent ack_received lost late_acked received decryption_failed);
```

Comment:
Please cast the types to `unsigned long long`. Otherwise, GCC on linux complains that it is not the same type as the format designator `%llu`. That is why you are seeing CI failures.

Ideally, we should be using `%" PRIu64 "` however including inttypes.h from bpftrace leads to other issues. Hence emit 64-bit types as long long.